src/Microsoft.AspNet.OData.Shared/Routing/Conventions/DynamicPropertyRoutingConvention.cs

```diff
@@ -18,7 +18,7 @@ namespace Microsoft.AspNet.OData.Routing.Conventions
     /// </summary>
     public partial class DynamicPropertyRoutingConvention
     {
-        private const string _actionName = "DynamicProperty";
```

Comment:
the recommended coding style is to add "_"  for the field?

Reply:
Not for private const.